libs/auth-nest-tools/README.md

```diff
@@ -20,12 +20,11 @@ Import and configure the AuthModule, example:
 @Module({
   imports: [
     AuthModule.register({
-      audience: 'protected_resource',
-      issuer: 'https://localhost:6001',
+      issuer: 'https://localhost:6001'
     }),
 ```
 
-where `audience` is the name your resource was registered under in IdS and `issuer` the IdS url.
+where `issuer` is the IdS url.
```

Comment:
We should also add a section that describes that the `ScopesGuard` is now required to verify at least one scope, so the API is verifying that the token is intended for this service. Otherwise it would be authorising all valid IDS tokens, intended other services.